lib/components/narrative/line-itin/realtime-time-column.js

```diff
@@ -29,26 +29,27 @@ function RealtimeTimeColumn ({
   timeOptions
 }) {
   const time = isDestination ? leg.endTime : leg.startTime
-  const formattedTime = time && formatTime(time, timeOptions)
+  // const formattedTime = time && formatTime(time, timeOptions)
   const isTransitLeg = isTransit(leg.mode)
   const isRealtimeTransitLeg = isTransitLeg && leg.realTime
-
   // For non-transit legs show only the scheduled time.
   if (!isTransitLeg) {
-    return <div>{formattedTime}</div>
+    return <>
```

Comment:
do we need `<>` if we already have a `<div>`?

Reply:
Removed in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/337d3b0a77ffbf0322683910f3e1647f79dc4d6b